app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -149,7 +149,26 @@ public void onClick(View view) {
             public void onClick(View view) {
                 if (scienceLab.isConnected()) {
                     Double capacitance = scienceLab.getCapacitance();
-                    saveAndSetData(String.valueOf(capacitance), getString(R.string.capacitance_unit));
+                    DecimalFormat capacitanceFormat = new DecimalFormat("#.##");
+                    String Capacitance;
+                    String capacitanceUnit;
+                    if (capacitance < 1e-9) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-12));
+                        capacitanceUnit = "picoFarad";
+                    } else if (capacitance < 1e-6) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-9));
+                        capacitanceUnit = "nFarad";
+                    } else if (capacitance < 1e-3) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-6));
+                        capacitanceUnit = "microFarad";
```

Comment:
These are not units. Refer https://en.wikipedia.org/wiki/Orders_of_magnitude_(capacitance)